packages/block/src/block.ts

```diff
@@ -71,6 +71,14 @@ export class Block {
     return new Block(header, transactions, uncleHeaders)
   }
 
+  /**
+   * Alias for Block.fromBlockData() with initWithGenesisHeader set to true.
+   */
+  public static genesis(blockData: BlockData = {}, opts: BlockOptions = {}) {
+    opts = { ...opts, initWithGenesisHeader: true }
+    return Block.fromBlockData(blockData, opts)
+  }
```

Comment:
Ah, thought we can really keep this as simple as you proposed (or more or less), so optimally `Block.genesis(common)`, and everyone needing something adjusted can just use the `Block.fromBlockData()` variant?

Reply:
Yeah I wanted to too but in practice (in our test suite) it was helpful to accept custom overrides as well.